gulpfile.js

```diff
@@ -50,7 +47,19 @@ gulp.task('browserify', function() {
       var filteredList = files.filter(_.bind(checkFileExtension,this,".js"));
       bundleThis(filteredList);
     }});
+}
 
+// Dependency task
+gulp.task('init_module', function(callback) {
+  exec("git submodule update --init && cd html5-common && npm install && cd ..", function(err) {
+    if (err) return callback(err);
+    callback();
+  });
+});
+
+// Build All
+gulp.task('build', ['init_module'], function () {
+  browserify_fn();
```

Comment:
same question as video plugins repo - does this run the build even if the init fails?